src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -78,16 +78,6 @@ void PrintOnboardingCodes(const chip::SetupPayload & payload)
     {
         ChipLogError(AppServer, "Getting manual pairing code failed!");
     }
-
-    // Hand a copy of the payload to GetLongManualPairingCode.
-    if (GetLongManualPairingCode(manualPairingCode, chip::SetupPayload(payload)) == CHIP_NO_ERROR)
```

Comment:
Are there any other consumers of `GetLongManualPairingCode`?  Can we just remove it?

Reply:
sure